docs/content/api_core/api_installing.rst

```diff
@@ -41,6 +46,10 @@ Second (from the root of the simpeg repository)::
 
     python setup.py install
 
+.. attention:: Windows users
+	A common error when installing the setup.py is:
+	``Missing linker, needs MSC v.1500 (Microsoft Visual C++ 2008) Runtime Library``
+	The missing library can be found _here: https://www.microsoft.com/en-ca/download/details.aspx?id=29
```

Comment:
The missing library can be hound here_. 

.. _here: https://www.microsoft.com/en-ca/download/details.aspx?id=29